site/content/how-to/_index.md

```diff
@@ -1,9 +1,4 @@
 ---
-title: "How-To Guides"
-description:
+title: "How-to guides"
```

Comment:
Do we have style guidance on this? I saw "How to" yesterday.

Reply:
FWIW, I see "How To Guide" in the [Topic types and templates](https://github.com/nginxinc/docs/blob/main/templates/style-guide.md#topic-types-and-templates). Since we're doing sentence case for titles, should that be replaced by:

- How to guide
- How-to guide

Reply:
In context, this is metadata for a section of documentation, not a singular topic - it names a section.